cmd/root.go

```diff
@@ -44,25 +42,7 @@ It can be used to export, import or sync entities to Kong.`,
 // sflags appropriately.
 // This is called by main.main(). It only needs to happen once to the rootCmd.
 func Execute() {
-	var wg sync.WaitGroup
-	var err error
-	const threads = 2
-	wg.Add(threads)
-
-	ctx, cancel := context.WithCancel(context.Background())
-
-	go func() {
-		defer wg.Done()
-		sendAnalytics(ctx)
-	}()
-
-	go func() {
-		defer wg.Done()
-		defer cancel()
-		err = rootCmd.Execute()
-	}()
-
-	wg.Wait()
+	err := rootCmd.Execute()
```

Comment:
[optional]

Just realized that this `err` does not get printed anywhere. Shall we print it?

Reply:
Good catch, added, Could you take another look?